CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # [Unreleased]
 
+* Adding basic user functionality
```

Comment:
`FEATURE:` to put this automatically in the Changelog when we release?

Reply:
Would be good... I'll take a second look on that node CLI tool that generated Changelog from github PRs